tests/dummy/app/index/template.hbs

```diff
@@ -38,7 +38,7 @@
             </code>
           {{/p.matryoshka}}
 
-          <button onclick={{action "add" model.tokens}}>Add title</button>
+          <button type="button" onclick={{action "add" model.tokens}}>Add title</button>
```

Comment:
Let's use `on` here instead of `onclick`

Reply:
This would be updated in next PR together all other template errors, there's hundreds of them.

Reply:
Ah, kk. 👍